.drone.yml

```
@@ -4371,9 +4371,21 @@ steps:
     volumes:
       - name: rpmrepo
         path: /rpmrepo
```

Similar to the DEB stage below, you should also mount the in-memory `tmpfs` here so the key material doesn't end up on disk.
```suggestion
    volumes:
      - name: rpmrepo
        path: /rpmrepo
      # for in-memory tmpfs for key material
      - name: tmpfs
        path: /tmpfs
```

---

Thanks!  Added in 20b9177 since I needed to re-sign the file.  I'll squash this before merge -- but wanted to leave it separate for review purposes.